src/app/shared/interfaces/json-patch.ts

```diff
@@ -0,0 +1,5 @@
+export interface JsonPatch {
+    op: string;
+    path: string[];
+    value?: any;
+  }
```

Comment:
if you run lint before pushing you will see this failing, I suggest you that you setup your text editor correctly to get this errors while you code.